UnoDrive.sln

```diff
@@ -122,6 +122,8 @@ Global
 		{7E754D3A-5074-41E4-A78F-D823C5E33B61}.AppStore|x86.Build.0 = Debug|x86
 		{7E754D3A-5074-41E4-A78F-D823C5E33B61}.AppStore|x86.Deploy.0 = Debug|x86
 		{7E754D3A-5074-41E4-A78F-D823C5E33B61}.Debug|Any CPU.ActiveCfg = Debug|x86
+		{7E754D3A-5074-41E4-A78F-D823C5E33B61}.Debug|Any CPU.Build.0 = Debug|x86
+		{7E754D3A-5074-41E4-A78F-D823C5E33B61}.Debug|Any CPU.Deploy.0 = Debug|x86
```

Comment:
For some reason the UWP isn't added to build and deploy, not sure if it's something on my end,

Reply:
Typically when you create a UWP project Any CPU is not mapped to x86 or x64. Any CPU really shouldn't be an option at all for UWP as it isn't supported. In UWP you have to specifically build for x64, x86, ARM, etc.

Kind of annoying if you aren't familiar with it and not a bad thing to include